### DISCUSSION TITLE
#90 - Eclipse border-radius and #998235+#F3AC3C

### DISCUSSION THREAD

**Discussion by rafehimself** [cssbattle/forum#149] [Bugs reports]
Oh c'mon  ...
that bug is repetitive ... on that colors again and again, maybe its time to fix it ? 😆 
![image](https://user-images.githubusercontent.com/58576132/139248087-98b1921a-c220-470f-906e-c756a2b3475e.png)
![image](https://user-images.githubusercontent.com/58576132/139248240-87e90c9b-34fc-4d66-8c38-c6aff45d5ae5.png)

**Comment by Beowolve** [comment]
For me the the `<p>` tag works, but `<body>` always has 99.9% despite using the same values.

**Reply by rafehimself** [reply]
I made 💯  somehow in slightly different way even with Q instead of px, quite bizarre cuz that solution had 0 displacement
 when you slide over fast back and forth  to compare, so it's pretty perfect pixel alignment, anyway that bug is repetitive
#50, #44

**Reply by chinchang** [reply]
Hey @rafehimself If you could please mail us your 99.9% solution to contact@cssbattle.dev, we can have a look.

**Reply by rafehimself** [reply]
@chinchang sent

**Reply by gunnarbittersmann** [reply]
As I said before, bringing your matches from 99.9% to 100% is part of the game. So stop crying eventually. Fix your solution and don’t demand the checking algorithm to be fixed.

**Reply by rafehimself** [reply]
@chinchang so I think now that is some overall  bug with border-radius
cuz if you look here:
#41

`<p><style>*{background:#293462}body{height:150;width:50;margin:180 150;border-radius:0 42q 0 0;box-shadow:0 -25vw#FE5F55;-webkit-box-reflect:right}p{border-radius:50%;padding:15;float:left;margin:-40 15`


that code give 100%, but if I change border-radius to border-radius:0 42q; it's not 100% anymore
but that doesn't change anything in output since it's exactly the same color of element and background,
that doesn't make any sense, how that influence the score?
